### ISSUE TITLE
Remove magnet move and position alerts and replace with single alert on key release

### ISSUE THREAD

**Issue by jhung** [phetsims/faradays-law#120]
Currently, multiple alerts are fired whenever the magnet moves, which causes alerts to lag behind the auditory cues from the sonification.

To address this, replace the multiple magnet move and position alerts with a single alert fired on key release / mouse release.

Specifically, remove the following alerts:
* coil proximity
* location change
* coil exit
* coil entrance direction
* coil bump
* magnetic field description change alerts
* slide stop

Replace the above alerts with a single alert is to be given on key release / mouse release / sliding stopped.

**Key release / Slide stop alert template - Only 4 loop coil visible**

Sliding stopped(1)| |Magnet location|4 loop coil proximity| |Coil entrance direction(2)|Field description(3)
---|---|---|---|---|---|---|
Sliding stopped.|Magnet at|<p>top-left,</p><p>top-center,</p><p>top-right,</p><p>middle-left,</p><p>middle-right,</p><p>center,</p><p>bottom-left,</p><p>bottom-center,</p><p>bottom-right,</p>|<p>far from</p><p>close to</p><p>very close to</p><p>in</p>|4 loop coil.|<p>4 loop coil to the left.</p><p>4 loop coil to the right.</p>|Field line description updated.

(1) if alert was triggered by slide action stopping
(2) if a coil opening is directly left or right.
(3) if field lines are visible

Example phrases:
* "Magnet top-right of play area. Far from 4 loop coil."
* "Magnet middle-left of play area. Very close to 4 loop coil to the right."
* "Magnet center of play area. In 4 loop coil. Field line description updated."
* "Sliding stopped. Magnet top-left of play area. Far from 4 loop coil. Field line description updated."

**Key release / Slide stop alert template - 2 and 4 loop coils visible**

Sliding stopped(1)| |Magnet location|4 loop coil proximity| |Coil entrance direction(2)|2 loop coil proximity| |Coil entrance direction(2)|Field description(3)
---|---|---|---|---|---|---|---|---|---|
Sliding stopped.|Magnet at|<p>top-left,</p><p>top-center,</p><p>top-right,</p><p>middle-left,</p><p>middle-right,</p><p>center,</p><p>bottom-left,</p><p>bottom-center,</p><p>bottom-right,</p>|<p>far from</p><p>close to</p><p>Very close to</p><p>In</p>|4 loop coil.|<p>4 loop coil to the left.</p><p>4 loop coil to the right.</p>|<p>Far from</p><p>Close to</p><p>Very close to</p><p>In</p>|2 loop coil|<p>2 loop coil to the left.</p><p>2 loop coil to the right.</p>|Field line description updated.

(1) if alert was triggered by slide action stopping
(2) if a coil opening is directly left or right. Due to the nature of the sim, the Entrance Description fragment appears only once since the magnet can not be to the right / left of both coils at the same time.
(3) if field lines are visible

Example phrases
* "Magnet top-left of play area. Far from 4 loop coil. Close to 2 loop coil. Field line description updated."
* "Magnet middle-left of play area. Close to 4 loop coil. Very close to 2 loop coil to the right. Field line description updated."
* "Sliding stopped. Magnet bottom-right of play area. Far from 4 loop coil. Far from 2 loop coil. Field line description updated."

**Comment by mbarlow12** [comment]
@jhung We're currently using aria-live regions for the interactive alerts and we update the PDOM for dynamic descriptions. I'm happy to explain a bit more if necessary.

Thoughts on the above: I like removing 'of play area', and I'm not sure how to handle the coil alerts. Would it be a bad idea to have some more detail in how we describe the coil proximity? Perhaps as the magnet gets near enough to one of the coils, we stop describing updates on the other? Stopping 'very close' to the 2-loop coil would thus result in:

'Magnet and center-top, very close to 2-loop coil, Field lines description updated'

This may also allow us to condense similar descriptions so that 
  'Far from 4-loop coil, far from 2-loop coil' becomes 'Far from both coils'
This may involve us redefining how the coil distance corresponds to the text.

The last suggestion is to shorten 'Field line description updated' to 'New field strengths at coil(s)', though I really don't know how users would respond to that.

**Comment by jhung** [comment]
> Would it be a bad idea to have some more detail in how we describe the coil proximity? Perhaps as the magnet gets near enough to one of the coils, we stop describing updates on the other?

I like this approach. Perhaps if the magnet is in the "Very close to coil" and "In the coil" regions, then we can stop describing the play area location all together.

> This may also allow us to condense similar descriptions so that
'Far from 4-loop coil, far from 2-loop coil' becomes 'Far from both coils'

This is good! Any opportunity to save a few words without losing clarity is a good thing.

> "New field strengths at coil(s)"

Perhaps "Field strength changed at coil(s)"? Also I am unsure how users will respond to that.

I'm concerned about the queuing of alerts if the user repeatedly taps a direction key. Maybe a delay would address this?

For example:

* **Focus magnet** "Magnet at top-right. W A S D keys moves magnet."
* **While holding or tapping A-key or down arrow** ... [ sonification sounds ] ...
* **Key released**
* **After 1 second delay, fire alert:** "Magnet at middle-right close to 4 loop coil. 4 loop coil to the  left." (note this alert has the slightly changed alert structure to make the coil direction more clear.)

**Comment by jhung** [comment]
I've updated the original issue:
* Removed "of play area" from alerts. i.e. "Magnet at [location] of play area... " now "Magnet at [location]..."
* Coil proximity now in the same sentence as magnet location.
* Coil entrance direction now its own sentence for clarity.

**Comment by jhung** [comment]
We should put this on hold pending discussions with @terracoda and @emily-phet regarding an explicit magnet pick-up and release interaction.

**Comment by jhung** [comment]
No longer applicable with revised designs. Closing this issue.
